src/components/questions/answersList.js

```diff
@@ -8,20 +8,47 @@ class AnswerList extends React.Component{
       answers: null,
       display: 2
     };
+    this.determineDisplayed = this.determineDisplayed.bind(this);
+    this.seeMoreAnswersClick = this.seeMoreAnswersClick.bind(this);
   }
 
-  makeAnswerArray(answerObj){
+  //DETERMINES NUMBER OF ANSWERS RENDERED
+  determineDisplayed(propsAnswers){
+    let keys = Object.keys(propsAnswers);
+    keys = keys.slice(0, this.state.display);
+    return keys;
+  }
 
+  //SETS NUMBER OF ANSWERS DISPLAYED TO ALL ON CLICK
+  seeMoreAnswersClick(event){
+    event.preventDefault();
+    this.setState({display: this.props.answers.length});
   }
 
   render(){
+    let keys = this.determineDisplayed(this.props.answers);
+
     return (
-      <div>
+      <div className="container">
       {
-      Object.keys(this.props.answers)[0] ? (Object.keys(this.props.answers).map( (key, index) => {
-        return <Answer key={index} id={this.props.answers[key].id} body={this.props.answers[key].body} date={this.props.answers[key].date} answerer_name= {this.props.answers[key].answerer_name} helpfulness={this.props.answers[key].helpfulness} photos={this.props.answers[key].photos}/>
+      keys[0] ?
+      (keys.map( (key, index) => {
+        return (
+        <Answer key={index}
+                id={this.props.answers[key].id}
+                body={this.props.answers[key].body}
+                date={this.props.answers[key].date}
+                answerer_name= {this.props.answers[key].answerer_name}
+                helpfulness={this.props.answers[key].helpfulness}
+                photos={this.props.answers[key].photos}
+        />)
```

Comment:
Would it be easier/cleaner to just pass down the whole answer (this.props.answers[key]) to the Answer component? Then you could destructure { answer } there and only have to access answer.id, answer.body, etc.